CMakeLists.txt

```diff
@@ -280,19 +280,21 @@ if(SECP256K1_BUILD_CTIME_TESTS)
   unset(msan_enabled)
 endif()
 
-set(SECP256K1_APPEND_CFLAGS "" CACHE STRING "Compiler flags that are appended to the command line after all other flags added by the build system. This variable is intended for debugging and special builds.")
-if(SECP256K1_APPEND_CFLAGS)
-  # Appending to this low-level rule variable is the only way to
+if(CMAKE_GENERATOR MATCHES "Make|Ninja")
+  # Only command-line build tool generators support low-level rule
+  # variables.
```

Comment:
nit:

While this seems to be true, I think it's a coincidence that these happen to be the command-line tools.
```suggestion
# Only some generators use low-level rule variables.
if(CMAKE_GENERATOR MATCHES "Make|Ninja")
```

And I think the command should be above the `if` line because it comments on the `if`.